lookbook/docs/patterns/05-dialogs.md.erb

```diff
@@ -39,7 +40,7 @@ class TestController < ApplicationControler
 end
 ```
 
-For this to work, the `OpTurbo::Streamable` module needs to be included in the modal component, along with the `OpTurbo::DialogStreamHelper` module in the corresponding controller:
+For this to work, the `OpTurbo::Streamable` module needs to be included in the modal component, along with the `OpTurbo::DialogStreamHelper` and `OpTurbo::ComponentStream` modules in the corresponding controller:
```

Comment:
I don't really like implicit dependencies between mixins. If we decide on keeping the two separate modules, then perhaps it'd be more straightforward to include `OpTurbo::DialogStreamHelper` in `OpTurbo::ComponentStream`.